juju/bundle.py

```diff
@@ -156,311 +171,663 @@ def resolve(self, reference):
                 reference = ref
         return reference
 
-    async def addCharm(self, change_id, charm, series):
+
+def get_charm_series(path):
```

Comment:
is this something that's bundle specific? Should it be in charm or a util section?

Reply:
I think this is just a bundle handler thing, was already existing in the codebase.